dGame/dComponents/PropertyManagementComponent.cpp

```diff
@@ -498,15 +498,15 @@ void PropertyManagementComponent::DeleteModel(const LWOOBJID id, const int delet
 		if (spawner != nullptr) {
 			dZoneManager::Instance()->RemoveSpawner(spawner->m_Info.spawnerID);
 		} else {
-			model->Smash(SILENT);
```

Comment:
this was the only "functional" change, since it was an error for it to be like this in the first place

Reply:
I care more about why it's an error than it being an error

Reply:
Smash takes (LWOOBJID, ekilltype), not just (ekilltype) as it was previously, it was before having undefined behavior